pycompwa/data/create.py

```diff
@@ -0,0 +1,52 @@
+"""Create an empty `pandas.DataFrame` according to the `~.PwaAccessor`."""
+
+
+__all__ = [
+    'empty_frame',
+    'multicolumn',
+]
+
+
+import pandas as pd
+
+from . import _labels
+from . import naming
+
+
+def empty_frame(particle_names: list = None,
```

Comment:
I don't understand the need for this function

Reply:
Please see [`data_module.ipynb`](https://github.com/redeboer/pycompwa/blob/9b9bece72007eabc9af1f0d485096c1694888a1e/examples/tools/data_module.ipynb)

Reply:
Slight modification btw:
https://github.com/ComPWA/pycompwa/pull/78/commits/fe47bb462a184247312d33baed6a66dd65ba18b7#diff-caa443b3abdcb2a1e840b380e9ad214eR57

Reply:
Ok, I see the use case now. Also like the rename